code/modules/martial_arts/judo.dm

```diff
@@ -0,0 +1,64 @@
+/datum/martial_art/judo
+	name = "Corporate Judo"
+	has_explaination_verb = TRUE
+	no_baton = TRUE
+	combos = list(/datum/martial_combo/judo/discombobulate, /datum/martial_combo/judo/eyepoke, /datum/martial_combo/judo/judothrow, /datum/martial_combo/judo/armbar, /datum/martial_combo/judo/wheelthrow)
+	weight = 5 //takes priority over boxing and drunkneness, less priority than krav or CQC/carp
+
+//Corporate Judo Belt
+
+/obj/item/judobelt
+	name = "Corporate Judo Belt"
+	desc = "Teaches the wearer NT Corporate Judo."
+	icon = 'icons/obj/clothing/belts.dmi'
+	lefthand_file = 'icons/mob/inhands/equipment/belt_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/belt_righthand.dmi'
+	icon_state = "judobelt"
+	item_state = "judo"
+	slot_flags = SLOT_BELT
+	var/datum/martial_art/judo/style
+
+/obj/item/judobelt/Initialize()
+	. = ..()
+	style = new()
```

Comment:
Might want a destroy that qdel_null's the style

Reply:
This is done in the style.remove line further down in the drop item code. Its treated the same way as krav currently. I really wouldn't want to touch that in this PR due to the possibility of breaking krav.